src/classes/RD2_OpportunityService.cls

```diff
@@ -149,9 +149,10 @@ public with sharing class RD2_OpportunityService {
     */
     private String buildOpportunityName(npe03__Recurring_Donation__c rd, Date closeDate) {
         return rd.npe03__Donor_Name__c + ' ' +
-            (rd.npe03__Amount__c != null ? rd.npe03__Amount__c.format() : '0') + ' ' +
-            (closeDate != null ? closeDate.format() : '') + ' ' +
-            UTIL_Describe.getObjectLabel('npe03__Recurring_Donation__c');
+            System.Label.npe03.RecurringDonationPrefix + ' (' +
+            (rd.npe03__Total_Paid_Installments__c != null ?
+            rd.npe03__Total_Paid_Installments__c + 1 : 1) + ') ' +
```

Comment:
The logic here that checks for a null value in `npe03__Total_Paid_Installments__c`, etc. already exists on line 134 where the code sets the `Recurring_Donation_Installment_Number__c` field. I recommend just using the value in that field vs. repeating the code. That would allow us to eventually improve the logic that sets that field and have it automatically apply to the naming logic.